docs/instruments/LORISLogicParser.md

```diff
@@ -0,0 +1,143 @@
+|-----------	|--------	|--------------------------------------------------------------------   |
+| add       	| a + b  	|                                                                   	|
+| subtract  	| a - b  	|                                                                    	|
+| negate    	| - a    	|                                                                    	|
+| multiply  	| a * b  	|                                                                     	|
+| divide    	| a / b  	| cannot divide by 0                                                	|
+| exponent  	| a ^ b  	|                                                                     	|
+| percentage  	| a %   	| divides the value of a by 100                                         |
+| factorial  	| a !   	| returns a factorial<br>supports 0 or positive numbers divisible by 0.5  |
+
+### Boolean/Comparison Operations (returns true or false)
+| Operation        	| Syntax  	| Notes                            	|
+|------------------	|---------	|----------------------------------	|
+| equivalency      	| a = b   	|                                  	|
+| inequivalency    	| a <> b  	|                                  	|
+| greater than     	| a > b   	|                                  	|
+| less than        	| a < b   	|                                  	|
+| greater or equal 	| a >= b  	|                                  	|
+| less or equal    	| a <= b  	|                                  	|
+| and              	| a and b 	| returns true if a and b are true 	|
+| or               	| a or b  	| returns true if a or b is true   	|
+| not              	| not a   	| returns true if a is false       	|
+
+### If Logic
+| Operation 	| Syntax       	| Notes                                                                            	|
+|-----------	|--------------	|----------------------------------------------------------------------------------	|
+| if        	| if(cond,x,y) 	| If the condition 'cond' evaluates to true, x is returned. If not, y is returned. 	|
+
+### Functions
+| Operation          	| Syntax               	| Notes                                                                                	|
+|--------------------	|----------------------	|--------------------------------------------------------------------------------------	|
+| not a number       	| isNaN(a)             	| returns true if a is not a number                                                    	|
+| modulo             	| mod(a, b)            	| returns the remainder of a / b                                                       	|
+| round              	| round(a, b)          	| rounds a to b decimal places; note all rounding functions support trailing 0s        	|
+| round up           	| roundup(a, b)        	| rounds a up to b decimal places                                                      	|
```

Comment:
"ceil" would be more consistent with most other languages (and "floor" instead of "rounddown")